astropy/io/fits/header.py

```diff
@@ -746,10 +746,6 @@ def tofile(self, fileobj, sep='', endcard=True, padding=True,
 
             if not fileobj.simulateonly:
                 fileobj.flush()
-                try:
-                    offset = fileobj.tell()
```

Comment:
I wonder what this is/was for. Can we safely delete this?

Reply:
Hard to say for sure, this comes from a commit eight years ago: https://github.com/spacetelescope/PyFITS/commit/f9f2be1175265e8db5d83083dceaa36e3c6a3c16, and I think that this was copied from another function (`_writeheader`) but there is no need to know the offset here, and as tests pass I think it is fine to remove. 
If at some point something fail because of this, then we will know why it was needed ;)